SU-HSDO/suhumsci
# fix(shs-5091): adding a check for if a heading tag is already present

# READY FOR REVIEW

## Summary
This PR adds a check for if an `<h>` tag is already present before printing an `<h3>` wrapper for the View title. 

## Urgency
medium

## Steps to Test
1. Visit a page like https://bingschool.suhumsci.loc/news-events/bing-times
2. Ensure that there is no longer an HTML error on "2023 Issue" title (halfway down the page) where the template prints an open `<h3>` tag, self-closes, and then the `<h2>2023 Issue</h2>` prints after it. 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
